src/index.jsx

```diff
@@ -33,10 +35,20 @@ subscribe(APP_INIT_ERROR, (error) => {
   );
 });
 
+subscribe(APP_CONFIG_INITIALIZED, () => {
+  loadExternalScripts([BrandingFontLoader], {
+    config: getConfig(),
+  });
+});
+
 initialize({
   handlers: {
     config: () => {
-      mergeConfig(configuration);
+      mergeConfig({
+        configuration,
```

Comment:
[nit] It probably should be like
```suggestion
        ...configuration,
```